components/region_cache_memory_engine/src/range_manager.rs

```diff
@@ -79,6 +83,19 @@ pub struct RangeManager {
     evicted_ranges: BTreeSet<CacheRange>,
     // ranges that are cached now
     ranges: BTreeMap<CacheRange, RangeMeta>,
+
+    // `pending_ranges` contains ranges that will be loaded into the memory engine. At
+    // sometime in the apply thread, the pending ranges, coupled with rocksdb snapshot, will be
+    // poped and pushed into `pending_ranges_with_snapshot`. Then the data in the snapshot
```

Comment:
the comment should be updated
```suggestion
    // poped and pushed into `ranges_loading_snapshot`. Then the data in the snapshot
```